js/currency-converter.js

```diff
@@ -0,0 +1,1027 @@
+var conversion_data = {
```

Comment:
This object is very large and makes it very difficult to navigate/reason about the file. I think it would be better to extract it to a separate JS file and then concatenate the two with GruntJS.